### ISSUE TITLE
Box, Snaxi, and Synergy Do Not Get Extra Insulated Gloves in Vendor

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#28923]
Deff and Metaclub are mapped to include 2 insulated gauntlets in the YouTool

**Comment by Kurfursten** [comment]
"Outpost" is just a hardmapped name, both of those engi-vends have no access requirements

Youtools never have access requirements

**Comment by drawsstuff** [comment]
This is incorrect, there is a difference between outpost youtools and engie-vends beyond name and access.

**Comment by Kurfursten** [comment]
Apparently this is var edited onto Deff (but it's not a real subtype)
![image](https://user-images.githubusercontent.com/9782036/109201281-48b39e80-7767-11eb-8e1a-517238aa8be8.png)

**Comment by kernikov** [comment]
Synergy has /obj/item/clothing/gloves/fyellow = 2.
